chore: Update build-essential to 12.12+deepin2

Avenger-285714 opened the pull request:
Link: http://deb.debian.org/debian/pool/main/b/build-essential/build-essential_12.12.tar.xz

Commit:
chore: Update build-essential to 12.12

Link: http://deb.debian.org/debian/pool/main/b/build-essential/build-essential_12.12.tar.xz
Signed-off-by: WangYuli <wangyuli@uniontech.com>

Zeno-sole commented:
@Avenger-285714  unresolvable: nothing provides g++ >= 4:14.2, (got version 4:13.2.0+really12.3.0-0deepin1)   请修改构建依赖版本为12

Commit:
fix: Downgrade dependencies on gcc and g++ to 12.3

Due to certain circumstances, we must ensure that GCC 12.3 is supported,
regardless of other factors.

Signed-off-by: WangYuli <wangyuli@uniontech.com>

Zeno-sole commented:
/integrate

Commit:
fix: Downgrade dependencies on dpkg-dev to 1.22.6

Due to certain circumstances, we must ensure that dpkg-dev 1.22.6 is
supported, regardless of other factors.

Signed-off-by: WangYuli <wangyuli@uniontech.com>
